### DISCUSSION TITLE
Transaction severity 'critical' not showing a warning dialog

### DISCUSSION THREAD

**Discussion by ian-eulith** [MetaMask/snaps#2102] [Q&A] [answered]
I am working on a snap that uses the transaction insights feature to show users information about proposed transactions. I am trying to use the new 'critical' transaction severity, but contrary to the description in [SIP-11](https://metamask.github.io/SIPs/SIPS/sip-11) returning `severity: 'critical'` alongside the UI content does not trigger any kind of warning in MetaMask. I am using version 1.3.2 of `@metamask/snaps-sdk` in my snap and MetaMask Flask version `11.7.0-flask.0`.

**Comment by Montoya** [comment, marked as answer]
What happens when you attempt to submit the transaction? To be safe, you can try it on a testnet.

**Reply by ian-eulith** [reply]
Ah, the warning shows up after I click 'confirm'. I expected it to show up prior, but that workflow makes sense to me too. Thanks!

**Comment by AlexeyKrasnoperov** [comment]
Hi Christian @Montoya,
I noticed that Transaction Insights severity: "critical" doesn't display a modal with the warning before the user can confirm the transaction in the latest Metamask Flask (13.2.0-flask.0). When I click 'Confirm', the transaction executes without displaying a warning. It previously showed the modal, but I'm unsure when the behavior changed, as I hadn't used the feature for quite some time.

**Reply by AlexeyKrasnoperov** [reply]
@Montoya should I create a separate discussion with the same topic?
I re-checked the scenario in the newest 13.4.2-flask.0, it doesn't work as well.

**Reply by Montoya** [reply]
This feature was dropped as part of the redesign of MetaMask confirmations and as of now there is no timeline to get it working. I will update the docs to clarify this.
